packages/web/src/components/Header/README.md

```diff
@@ -0,0 +1,279 @@
+# Header
+
+> Header wraps top-level navigation and branding.
+
+Header is a highly variable and customizable component. It comes in several
+design variants and provides a handful of building blocks you can use to achieve
+your specific design goals.
+
+## No JavaScript Provided!
+
+⚠️ No JavaScript is currently bundled with the Header component, you need to
+write the controlling scripts yourself. However, feel free to inspire by our
+[demo in Spirit examples][examples].
+
+## Accessibility Guidelines
+
+- Ensure accessibility by using a `<nav>` element or, if using a more generic
+  element such as a `<div>`, add a `role="navigation"` to every navigation to
+  explicitly identify it as a landmark region for users of assistive
+  technologies.
+- Indicate the current item by using `aria-current="page"` for the current
+  page or `aria-current="true"` for the current item in a set.
+- Don't forget to provide textual equivalents for icon buttons. Use either the
+  `aria-label` attribute, or the provided `accessibility-hidden` helper class.
+- For Header to work correctly with assistive technologies, the `aria-expanded`
+  attributes need to be updated by JavaScript when Header is toggled.
+
+👉 The animation effect of this component is dependent on the
+`prefers-reduced-motion` media query.
+
+## Minimal Header
+
+Without any modifier, Header is ready to contain necessary blocks in a classic
```

Comment:
```suggestion
Without any modifier, the Header is ready to contain necessary blocks in a classic
```

Reply:
To be sure, I checked the whole text with Grammarly and fixed all errors.